pkg/binlog/reader/util_test.go

```diff
@@ -1,93 +0,0 @@
-// Copyright 2020 PingCAP, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package reader
-
-import (
-	"context"
-
-	gmysql "github.com/go-mysql-org/go-mysql/mysql"
-	"github.com/go-mysql-org/go-mysql/replication"
-	. "github.com/pingcap/check"
-
-	"github.com/pingcap/dm/pkg/utils"
-)
-
-// added to testTCPReaderSuite to re-use DB connection.
-func (t *testTCPReaderSuite) TestGetGTIDsForPos(c *C) {
```

Comment:
seems we can not test this func without a connection to mysql  and we actually tested these functions in many integration tests, so I just removed these two test cases